sdk/storage/test-resources.json

```diff
@@ -21,6 +21,13 @@
             "metadata": {
                 "description": "The principal to assign the role to. This is application object id."
             }
+        },
+        "enableStorageVersioning": {
+          "type": "bool",
+          "metadata": {
+            "description": "Enable Versioning enabled setting, default value is true"
+          },
+          "defaultValue": true
         }
```

Comment:
What's the reason for this? We already have `versionedAccountName` in the account set used in this arm template.
Some tests in the suite rely on versioning enabled and some on versioning enabled.

Reply:
This change has been rolled back